### PR TITLE
Change/jetpack contact form icon

### LINKED ISSUE #26865: Jetpack: Contact Form - Update block icon
Current icon

Image

### PR DESCRIPTION
**Pull request by CGastrell**
Wee change: replace Jetpack Contact Form icon

Fixes #26865 

#### Changes proposed in this Pull Request:
- replace Jetpack Contact Form icon with the one provided

Before:
image

After:
image

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Apply patch and insert a `/form`, you should see the new icon:
image

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/contact-form/index.js [hunk 1]
```diff
@@ -20,16 +20,18 @@ import variations from './variations';
 export const name = 'contact-form';
 
 const icon = renderMaterialIcon(
-	<Path d="M13 7.5h5v2h-5zm0 7h5v2h-5zM19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16H5V5h14v14zM11 6H6v5h5V6zm-1 4H7V7h3v3zm1 3H6v5h5v-5zm-1 4H7v-3h3v3z" />
+	<Path
+		fill-rule="evenodd"
+		clip-rule="evenodd"
```
Comment (oskosk) [inline projects/plugins/jetpack/extensions/blocks/contact-form/index.js]: These ones need to be in camel case (React demands it)

**Comment by oskosk** [comment]
Pushed renaming ofprops clip-rule/fill-rule to clipRule and fillRule in [5b894a9](https://github.com/Automattic/jetpack/pull/27010/commits/5b894a9131c00d0956a5db72f9e00d3a43046602)

File: projects/plugins/jetpack/extensions/blocks/contact-form/index.js (hunk 1 above)
Comment (oskosk) [reply projects/plugins/jetpack/extensions/blocks/contact-form/index.js]: Addressed in [5b894a9](https://github.com/Automattic/jetpack/pull/27010/commits/5b894a9131c00d0956a5db72f9e00d3a43046602)

**Comment by pablohoneyhoney** [comment]
Are there other iterations of this icon? And are the lines 1.5 thick?

**Review by jcheringer — APPROVED** [review]
Works for me!

**Comment by digitalwaveride** [comment]
@pablohoneyhoney 

> Are there other iterations of this icon? And are the lines 1.5 thick?

Lines are 1.5:

image

Other icon iterations:

image


In context:

image

**Review by samiff — APPROVED** [review]

**Pull request merged**